deployments/engine/docker-compose/config/master_with_s3.toml

```diff
@@ -0,0 +1,27 @@
+[framework-meta]
+endpoints = ["mysql-standalone:3306"]
+schema = "test_framework"
+user = "root"
+password = ""
+
+[business-meta]
+endpoints = ["mysql-standalone:3306"]
+schema = "test_business"
+user = "root"
+password = ""
+
+[log]
+level = "debug"
+
+[job-backoff]
+reset-interval = "2s"
+initial-interval = "1s"
+max-interval = "15s"
+max-try-time = 100
+
+[storage.s3]
+bucket = "engine-it"
+endpoint = "http://minio-standalone:9000/"
+access-key = "engine"
+secret-access-key = "engineSecret"
+force-path-style = true
```

Comment:
Do we always set `force-path-style` to true, if so what about setting it in code instead of configuration file.